docs/core/diagnostics/distributed-tracing-builtin-activities.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Built-in Activities in .NET
+description: Overview of activities emitted by the .NET libraries
+ms.date: 12/05/2024
+ms.topic: reference
+---
+
+# Built-in activities in .NET
+
+This is a reference for distributed tracing [activities](xref:System.Diagnostics.Activity) emitted natively by .NET's built-in <xref:System.Diagnostics.ActivitySource> instances.
+
+## System.Net activities
+
+### HTTP client request
+
+<xref:System.Net.Http.SocketsHttpHandler> and <xref:System.Net.Http.HttpClientHandler> report the HTTP client request activity following the recommendations defined in OpenTelemetry [HTTP Client Semantic Conventions](https://opentelemetry.io/docs/specs/semconv/http/http-spans/#http-client). It describes the HTTP request sent by <xref:System.Net.Http.HttpClient>'s send overloads during the time interval the underlying handler completes the request. Completing the request includes the time up to reading response headers from the network stream. It doesn't include the time spent reading the response body.
```

Comment:
> unless the underlying handler does so before returning the response

Since `HttpClientHandler` internal chain is currently private, we can expect only `SocketsHttpHandler` and platform handlers here. Are you aware of cases when any of those would buffer the content?

Reply:
The Wasm client does buffer by default (https://github.com/dotnet/runtime/issues/77904#issuecomment-2509592494), but I'm not sure we care about OTEL tracing there?

Reply:
> I'm not sure we care about OTEL tracing

Rather not than yes and I think the WASM behavior is problematic, we should consider fixing instead of complicating our docs.